### PR TITLE
Small consistencies across the Settings section of the Dashboard.

### PR DESCRIPTION
**Pull request by dburka**
**Story card:** None

## Because

A bunch of small inconsistencies were driving me crazy. I spent an hour doing clean-up over the weekend.

## This addresses

Small things like buttons colors, breadcrumb navs, some weird headers, adding a helpful form tip, etc. Note that protocol drugs are now listed alphabetically... check my code there, I took a simple thing from StackOverflow.

## Test instructions

This should all be cosmetic.

### CODE REVIEW

File: app/views/admin/protocols/show.html.erb [hunk 1]
```diff
@@ -13,17 +13,17 @@
   </div>
   <nav class="page-nav">
     <% if current_admin.accessible_organizations(:manage).any? || current_admin.power_user? %>
-      <%= link_to "Edit protocol", edit_admin_protocol_path(@protocol), class: "btn btn-sm btn-primary", id: "Edit protocol" %>
-      <%= link_to 'New protocol drug', new_admin_protocol_protocol_drug_path(@protocol), class: "btn btn-sm btn-primary", id: "New protocol drug" %>
+      <%= link_to "Edit protocol", edit_admin_protocol_path(@protocol), class: "btn btn-sm btn-outline-primary", id: "Edit protocol" %>
+      <%= link_to 'New protocol drug', new_admin_protocol_protocol_drug_path(@protocol), class: "btn btn-sm btn-success", id: "New protocol drug" %>
```
Comment (claudiovallejo) [inline app/views/admin/protocols/show.html.erb:17]: @dburka can we use the '+' instead of 'New' to keep the green buttons consistent across pages?

Comment (dburka) [reply app/views/admin/protocols/show.html.erb:17]: Yes @claudiovallejo , we can

Comment (claudiovallejo) [reply app/views/admin/protocols/show.html.erb]: I pushed the change @dburka

**Review by claudiovallejo — APPROVED** [review]
Looks good to me ✅

**Pull request merged**